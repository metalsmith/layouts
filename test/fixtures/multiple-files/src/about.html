---
layout: standard.hbs
---
<h1>The about page</h1>
