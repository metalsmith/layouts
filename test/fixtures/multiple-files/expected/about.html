<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <h1>The about page</h1>

  </body>
</html>
